orca-queue/src/main/kotlin/com/netflix/spinnaker/config/QueueConfiguration.kt

```diff
@@ -63,5 +72,13 @@ open class QueueConfiguration {
       // TODO: should set an error handler as well
     }
 
-  @Bean open fun applicationEventTaskExecutor(): ThreadPoolTaskExecutor = ThreadPoolTaskExecutor()
+  @Bean open fun applicationEventTaskExecutor(registry: Registry): ThreadPoolTaskExecutor =
+    applyThreadPoolMetrics(
+      registry,
+      ThreadPoolTaskExecutor().apply {
+        corePoolSize = 20
+        maxPoolSize = 20
+      },
+      "applicationEventMulticaster"
+    )
```

Comment:
This is the thread pool used to deliver application events via Spring's event bus